.github/workflows/ci.yaml

```diff
@@ -40,7 +40,9 @@ env:
   # - 10.6.10 is the version currently shipped with the Add-on (as of 31 Jan 2023)
   # 10.10 is the latest short-term-support
   # - 10.10.3 is the latest (as of 6 Feb 2023)
-  MARIADB_VERSIONS: "['mariadb:10.3.32','mariadb:10.6.10','mariadb:10.10.3']"
+  # mysql 8.0.32 does not always behave the same as MariaDB
+  # and some queries that work on MariaDB do not work on MySQL
+  MARIADB_VERSIONS: "['mariadb:10.3.32','mariadb:10.6.10','mariadb:10.10.3','mysql:8.0.32']"
```

Comment:
Should we rename everything?

Reply:
I'm not sure - if they behave differently then maybe it should be a completely two separate steps:

```yaml
  pytest-mariadb:
    runs-on: ubuntu-22.04
    services:
      mariadb:
        image: ${{ matrix.mariadb-group }}
        ...

  pytest-mysql:
    runs-on: ubuntu-22.04
    services:
      mysql:
        image: ${{ matrix.mysql-group }}
        ...
```

Reply:
The setup is the same.  I'm wondering if we should make it MariaDB-MySQL-versions?  Maybe not worth the churn?

Reply:
For me the most confusing thing is the `services` section:
```yaml
    services:
      mysql
```

If we are happy that it doesn't matter, then I don't think the name of the `MARIADB_VERSIONS` or `MARIADB_MYSQL_VERSIONS` variable is important.

Reply:
I think thats fine. If we want to rename the variables we can do so later. I think its more important to start testing this since we don't want any more regressions.